src/containers/AudioUploader/components/AudioContent.tsx

```diff
@@ -0,0 +1,111 @@
+/**
+ * Copyright (c) 2016-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import React, { Fragment } from 'react';
+import { injectT, tType } from '@ndla/i18n';
+import { connect, FormikContextType } from 'formik';
+import BEMHelper from 'react-bem-helper';
+import { UploadDropZone } from '@ndla/forms';
+import styled from '@emotion/styled';
+import Tooltip from '@ndla/tooltip';
+import { DeleteForever } from '@ndla/icons/editor';
+import IconButton from '../../../components/IconButton';
+import AudioPlayer from './AudioPlayer';
+import FormikField from '../../../components/FormikField';
+import { AudioFormikType } from './AudioForm';
+
+interface BaseProps {
+  classes: BEMHelper<BEMHelper.ReturnObject>;
+}
+
+interface Props extends BaseProps {
+  formik: FormikContextType<AudioFormikType>;
+}
+
+const StyledDeleteButtonContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+`;
+
+const getPlayerObject = (
+  values: AudioFormikType,
+): { src: string; mimeType: string } | undefined => {
+  const { newFile, storedFile } = values.audioFile;
+
+  if (newFile) {
+    return {
+      src: newFile.filepath,
+      mimeType: newFile.file.type,
+    };
+  } else if (storedFile) {
+    return {
+      src: storedFile.url,
+      mimeType: storedFile.mimeType,
+    };
+  }
+  return undefined;
+};
+
+const AudioContent = ({ t, formik }: Props & tType) => {
+  const { values, setFieldValue } = formik;
+  const PlayerOrSelector = () => {
+    const playerObject = getPlayerObject(values);
+    if (playerObject) {
+      return (
+        <>
+          <StyledDeleteButtonContainer>
+            <AudioPlayer audio={playerObject} />
+            <Tooltip tooltip={t('form.audio.remove')}>
+              <IconButton
+                onClick={() => {
+                  setFieldValue('audioFile', {});
+                }}
+                tabIndex={-1}>
+                <DeleteForever />
```

Comment:
syns det delete symbolet er litt merkelig plassert. Hadde det gitt mer mening om det kom rett bak lydfilen?
<img width="521" alt="Screenshot 2021-04-13 at 13 50 47" src="https://user-images.githubusercontent.com/10741877/114565219-5a96c580-9c71-11eb-905f-b5d1aec1f093.png">

Reply:
Jo enig, skal se om jeg klarer å lage det penere. 
(Forslag til hvordan man plasserer denne uten å slakte css tas i mot med takk :smile: )

Reply:
hmm, ser den boksen som wrapper de to har display: flex. Så første jeg tenkte var at du kan legge til en `align-items: center` eller `align-items: baseline` + enten større knapp eller noe `margin/padding` kunne fikset dette?

Reply:
https://css-tricks.com/snippets/css/a-guide-to-flexbox/ <3

Reply:
> hmm, ser den boksen som wrapper de to har `display: flex`. Så første jeg tenkte var at du kan legge til en `align-items: center` eller `align-items: baseline` + enten større knapp eller noe `margin/padding` kunne fikset dette?

Jaa jeg prøvde det, men de havner ganske off, så jeg testet med noe margin som ser ut til å funke bra, men kanskje hacky?

Reply:
glemte vist å svare deg, så lenge det ser oversiktlig ut så går det nok greit. Er ikke css som kommer til å bli endret så mange ganger.